pyproject.toml

```diff
@@ -12,6 +12,9 @@ exclude = [".github"]
 packages = ["bowtie"]
 
 [tool.hatch.build.targets.wheel.hooks.custom]
+dependencies = [
+  "requests"
```

Comment:
I think GitHub3.py would have possibly still saved you from implementing the pagination manually (it doesn't have a packages API but you can I think use `GitHubSession` to make a request to an arbitrary URL) -- but since you wrote it already I'm also fine with just using an HTTP client. But not requests, can you use `httpx` instead -- I think nothing should probably change, just replacing `requests` with `httpx` I think should more or less work the same.